config/source/multi/deployments/adapter.yaml

```diff
@@ -80,3 +80,18 @@ spec:
           containerPort: 8008
 
       terminationGracePeriodSeconds: 10
+      affinity:
```

Comment:
do we set this anywhere in Knative already?

Is that something that we might have the user/admin have control on? Or allow tweaking?

Reply:
[Yes (single zone only)](https://github.com/knative/eventing/blob/main/config/core/deployments/controller.yaml#L33) and yes, though it really depends on the available scheduling strategies.